src/Client/Web/Bit.Client.Web.BlazorUI/Components/Toggles/BitToggle.razor.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Web;
+
+namespace Bit.Client.Web.BlazorUI
+{
+    public partial class BitToggle
+    {
+        private bool isChecked;
+
+        [Parameter] public string OnText { get; set; }
+        [Parameter] public string OffText { get; set; }
+        [Parameter] public bool IsInlineLabel { get; set; }
+
+        [Parameter]
+        public bool IsChecked
+        {
+            get => isChecked;
+            set
+            {
+                if (value == isChecked) return;
+                isChecked = value;
+                ClassBuilder.Reset();
+                _ = IsCheckedChanged.InvokeAsync(value);
+            }
+        }
+
+        [Parameter] public EventCallback<bool> IsCheckedChanged { get; set; }
+
+
+        [Parameter] public RenderFragment ChildContent { get; set; }
+        [Parameter] public EventCallback<MouseEventArgs> OnChange { get; set; }
```

Comment:
The OnChange event argument's type should not be "MouseEventArgs", but the type of the value that is changing itself, which is "bool" in this case.